src/cfnlint/data/ExtendedSpecs/all/01_spec_patch.json

```diff
@@ -120,6 +120,11 @@
     "path": "/ResourceTypes/AWS::ResourceGroups::Group/Properties/Tags/ItemType",
     "value": "Tag"
   },
+  {
+    "op": "replace",
+    "path": "/ResourceTypes/AWS::IoT::ProvisioningTemplate/Properties/Tags/ItemType",
+    "value": "Tag"
+  },
```

Comment:
leaving a comment just to call out this part is hand-written / not generated by the scripts